front/src/modules/command-menu/components/CommandMenu.tsx

```diff
@@ -200,6 +200,28 @@ export const CommandMenu = () => {
             />
           ))}
         </CommandGroup>
+        <CommandGroup heading="Create">
```

Comment:
@brendanlaschke shouldn't we fix the code line 136 instead?

Reply:
> @brendanlaschke shouldn't we fix the code line 136 instead?

We did it so we don't obstruct the search with the goto/create commands earlier (as they are not so relevant as the other search results) - but if its fine if they appear at the top sure this can be changed ...

Reply:
I would keep them in the same order yes, it's weird that they jump to the end when we search.
I've tried on Linear and they don't change it!